btree/btree.c

```diff
@@ -1544,6 +1544,24 @@ static struct m0_rwlock list_lock;
  */
 static int64_t lru_space_used = 0;
 
+/**
+ * Watermarks for BE space occupied by nodes in lru list.
+ */
+/** LRU purging should not happen below low used space watermark. */
+int64_t lru_space_wm_low    = 2 * 1024 * 1024 * 1024ULL;
```

Comment:
Assign the enum name M0_USW_LOW to this variable instead of hardcoded number which makes it easier to read.
```suggestion
int64_t lru_space_wm_low    = M0_USW_LOW;
```

Reply:
Done.